cha-ching-app/src/error.css

```diff
@@ -0,0 +1,35 @@
+.errorText {
+  color: #850f09;
+  display: flex;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 22px;
+  font-weight: 400;
+}
+
+.errorContainer {
+  align-items: center;
+  background-color: #ffe3f6;
+  border-radius: 20px;
+  border: 2px solid #850f09;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  height: 200px;
+  justify-content: center;
+  left: 0;
+  margin-left: auto;
+  margin-right: auto;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  top: 100px;
+  width: 326px;
+}
+
+.errorWrapper {
+  display: flex;
```

Comment:
Thought this was a cool article with different ways of showing visuals for errors in the app: https://blog.hubspot.com/blog/tabid/6307/bid/33766/10-clever-website-error-messages-from-creative-companies.aspx

Reply:
Great, i'll look into this for some stying inspiration.